LibationWinForms/BookLiberation/ProcessorAutomationController.cs

```diff
@@ -152,7 +141,7 @@ public static async Task BackupAllPdfsAsync(EventHandler<LibraryBook> completedA
 
             var downloadPdf = getWiredUpDownloadPdf(completedAction);
 
-			(AutomatedBackupsForm automatedBackupsForm, LogMe logMe) = attachToBackupsForm(downloadPdf);
```

Comment:
For everything in this file, I don't know if I'm more impressed that I made this mess able to be followed or if I'm more impressed that you were able to follow it. Remember last night's conversation about separation of concerns sometimes being a mess of injected methods and weird abstractions? Guilty

Reply:
Well I'll let you in on a little secret: reverse engineering is a hobby of mine ;-)

Reply:
> reverse engineering is a hobby of mine

It shows. Quality contributions.